packages/cli-kit/src/node/node-package-manager.ts

```diff
@@ -298,7 +298,6 @@ export interface DependencyVersion {
 export async function addNPMDependenciesIfNeeded(
   dependencies: DependencyVersion[],
   options: AddNPMDependenciesIfNeededOptions,
-  force = false,
```

Comment:
This `force` param is the gross hack that I wanted to get around. `IfNeeded` means this is a more specialized method, and we shouldn't have a `force` param which contradicts the `IfNeeded`.